test/report_data_test.py

```diff
@@ -0,0 +1,60 @@
+import uuid
+import unittest
+from datetime import datetime
+
+import src.report.report_data as report_data
+import src.manifest as company
+from src.database_context import DatabaseContext
+from src.bundle_info import BundleInfo
+from src.client_info import ClientInfo
+from src.user_dto import UserDto
+from src.subscription_info import SubscriptionInfo
+
+
+class ReportDataTests(unittest.TestCase):
+    def setUp(self) -> None:
+        self.db = DatabaseContext('db')
```

Comment:
Won't this use the same database that is used outside of tests? If yes, the tearDown method can delete important data after test completion.